src/main/java/ArticleSwitchPage.java

```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
```

Comment:
Please remove the files `.gradle/7.0/fileHashes/fileHashes.lock` and ` .gradle/buildOutputCleanup/buildOutputCleanup.lock `